tests/btc_math_tests.move

```diff
@@ -72,6 +72,12 @@ fun bits_to_target_tests() {
     let target = bits_to_target(bits);
     assert!(target == 0x00ffff0000000000000000000000000000000000000000000000000000000000);
     assert!(bits == target_to_bits(target));
+
+    let bits = 0x1c0168fd;
```

Comment:
```suggestion
   // https://learnmeabitcoin.com/explorer/block/0000000000519051eb5f3c5943cdbc176a0eff4e1fbc3e08287bdb76299b8e5c
    let bits = 0x1c0168fd;
```

Reply:
add info that it is a block where we compute the new dificutly